src/kv/kv.h

```diff
@@ -1184,6 +1210,11 @@ namespace kv
     template <class K, class V, class H = std::hash<K>>
     using Map = Map<K, V, H, S, D>;
     using Tx = Tx<S, D>;
+#ifdef PBFT
```

Comment:
I think the `KV` should not know about PBFT. Could the default behaviour be set in `networktables.h` instead?

Reply:
So this is a good question. The thing is that we do not only create tables in `networktables.h`. There are a lot of app tables to, which means we will have to check if we are using PBFT or not in a lot of places. It is not ideal for the KV to know about PBFT but this is the place where this change has the minimal effect (afaik)

Reply:
As discussed, one of the options would be to wrap the `Store` object that is available to any frontend (`tables`) and in `networktables.h` in an object that is responsible for creating the maps. Then, we can use this object to define the `replicated_default` for maps when they are created.

Reply:
As discussed, this is OK for now.

Reply:
Why does this need to be static? Why can't we set that on the KV at construction time? We want to move away from ifdef and towards a runtime switch for the consensus.

Reply:
As discussed, will modify this so that the consensus can pass either none, all, or a list of tables that it wants to replicate. Then the KV can set this bool for each table on creation.